packages/react-native/template/metro.config.js

```diff
@@ -1,11 +1,30 @@
-const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
+const {getDefaultConfig} = require('@react-native/metro-config');
 
 /**
  * Metro configuration
  * https://facebook.github.io/metro/docs/configuration
  *
  * @type {import('metro-config').MetroConfig}
  */
-const config = {};
+const config = getDefaultConfig(__dirname);
 
-module.exports = mergeConfig(getDefaultConfig(__dirname), config);
```

Comment:
Is this the way Metro team intends to merge configs? cc @huntie

Reply:
In discussions with Metro experts at Expo, the general feeling is that using `mergeConfig` can be tricky, so it's safer to get the default config and just make exactly the changes you need.

Reply:
Yeah it gets the job done. At the same time I'd like the core Metro team to be aware that the community needs to "hack" the official approach

Reply:
Using `mergeConfig` is totally optional — it's a convenience. However we provide the existing pattern because it lets users add their custom config inside the `config` variable as one object, avoiding procedural assignments. I would suggest this pattern be preserved for `react-native-tvos`, to avoid fragmentation.

In this case, it's close to the example we have in the [Configuring Metro](https://reactnative.dev/docs/metro#configuring-metro) docs guide. This would replace lines 4-6 from the below with your commented override.

<img width="707" alt="image" src="https://github.com/react-native-tvos/react-native-tvos/assets/2547783/90fa4ff8-3936-44ac-aaa0-e723ffd30250">

Note also that `sourceExts` are later dynamically extended by `@react-native/community-cli-plugin`. This might be another location to augment the user's input Metro config.

Reply:
Thanks @huntie! I had thought that by extending `@react-native/metro-config`, we pick up all the changes done by the community plugin. Is this not correct?